polaris-core/src/main/java/org/apache/polaris/core/context/CallContext.java

```diff
@@ -100,8 +100,9 @@ public Map<String, Object> contextVariables() {
    * CallContext}.
    */
   static CallContext copyOf(CallContext base) {
-    RealmContext realmContext = base.getRealmContext();
-    PolarisCallContext polarisCallContext = base.getPolarisCallContext();
+    String realmId = base.getRealmContext().getRealmIdentifier();
+    RealmContext realmContext = () -> realmId;
+    PolarisCallContext polarisCallContext = PolarisCallContext.copyOf(base.getPolarisCallContext());
```

Comment:
Nit: there are new comments in the caller, could we also add comments here to explain why we need cloned objects `realmContext` and `polarisCallContext`.

Reply:
@flyrain this will soon disappear with #589 so I don't think there is need to explain.

But just for posterity, the reason is realted to `TaskExecutor`: these beans are request-scoped, and they would become unusable when the original request that created them completes, but the task hasn't finished executing yet.